testsuite/testcases/src/state_sync_performance.rs

```diff
@@ -23,81 +21,86 @@ impl Test for StateSyncPerformance {
 
 impl NetworkTest for StateSyncPerformance {
     fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
```

Comment:
Maybe outside of scope of this PR but it'd be a lot easier if this was just an async function running under `tokio::test`, to avoid having to do all this runtime stuff.